endoreg_db/management/commands/setup_endoreg_db.py

```
@@ -0,0 +1,184 @@
+            "endoreg_db_aimodel",
+            "endoreg_db_modelmeta",
+            "django_cache_table",
+        ]
+
+        cursor = connection.cursor()
+        cursor.execute("SELECT name FROM sqlite_master WHERE type='table';")
+        existing_tables = [row[0] for row in cursor.fetchall()]
+
+        missing_tables = [table for table in required_tables if table not in existing_tables]
+        if missing_tables:
+            raise Exception(f"Missing required tables: {missing_tables}")
```

Ruff (0.14.2)

Create your own exception

(TRY002)